README.md

```diff
@@ -24,16 +24,18 @@ This Filelist can be placed anywhere, as long as FsGuard has access to it when i
 FsGuard expects a minisign signature and filelist to be appended to the binary. An example signature "set" can be found [here](https://github.com/linux-immutability-tools/FsGuard/blob/main/signatures).
 A signature set can be generated and added to FsGuard with these commands:
 ```bash
+#!/bin/sh
```

Comment:
I would remove the shebang, this isn't mean to be ran as a shell script

Reply:
Done